src/Trie.js

```diff
@@ -95,15 +89,15 @@ class Trie extends Map {
   }
 
   /**
-    Sets the value for the key in the Trie object. Returns the Trie object
+    Sets the value for the key in the Trie object. 
 
     @example store.set('Michael Jackson', { id: 1 })
     @example store.set('Lord Kelvin', 1824)
     @example store.set('William Osler', 'Scientist')
     @example store.set('Anton Webern', [])
     @example store.set('Charles Best', function info () {})
   */
-  set (key: string, value: any): Map {
+  set (key: string, value: any): void {
```

Comment:
About `set` return again

> Sets the value for the key in the Map object. Returns the Map object.

So the annotation was correct but the implementation isn't